### DISCUSSION TITLE
Emojis

### DISCUSSION THREAD

**Discussion by BobSanScans** [xulihang/ImageTrans-docs#460] [Q&A] [answered]
Anyone know how to put in emojis, I found one that worked, but is there a list somewhere I can find with emojis that will work with the software or an emoji insert I can use?

**Comment by xulihang** [comment, marked as answer]
I use Windows's virtual keyboard to input emojis.

https://github.com/xulihang/ImageTrans-docs/assets/5462205/a8eac796-571e-4fc3-8871-7733902c7318

**Reply by BobSanScans** [reply]
Wow did not expect such a quick reply, and a video showing exactly how to do it! Thanks a ton! Really love Imagetrans so far! Finally able to read dropped novels and mangas easily!
